### DISCUSSION TITLE
Convert OpenAssistant/oasst-sft-6-llama-30b-xor

### DISCUSSION THREAD

**Discussion by DawoodTouseef** [ggml-org/llama.cpp#3295] [Q&A]
I'm trying to convert to GGUF the new [OpenAssistant/oasst-sft-6-llama-30b-xor](https://huggingface.co/OpenAssistant/oasst-sft-6-llama-30b-xor/tree/main/oasst-sft-6-llama-30b-xor/oasst-sft-6-llama-30b-xor)

python3 convert.py /Downloads/phi-1_5 
Loading model file /Downloads/ooost/pytorch_model.bin
Traceback (most recent call last):
  File "/llama.cpp/convert.py", line 1196, in <module>
    main()
  File "/llama.cpp/convert.py", line 1145, in main
    params = Params.load(model_plus)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/llama.cpp/convert.py", line 299, in load
    params = Params.loadHFTransformerJson(model_plus.model, hf_config_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/llama.cpp/convert.py", line 209, in loadHFTransformerJson
    n_embd           = config["hidden_size"]
                       ~~~~~~^^^^^^^^^^^^^^^
KeyError: 'hidden_size'

**Comment by ianscrivener** [comment]
You ran `pip install -r requirements`?
What version of python are you using?
Personally I've had some issues with python 3.9.x & 3.11.x... so use python **3.10.12** as my default for AI & LLMs.

**Comment by KerfuffleV2** [comment]
Did you un-XOR it first? It looks like even the metadata like JSON files are XORed.

**Comment by DawoodTouseef** [comment]
Idk I Got it from hugging face

**Comment by KerfuffleV2** [comment]
> Idk I Got it from hugging face

Okay, then that's a "no". Please read the model card for the model you're talking about having issues with: https://huggingface.co/OpenAssistant/oasst-sft-6-llama-30b-xor

It's basically encrypted. You'll need to follow the instructions there to decrypt it.
